test/span.js

```diff
@@ -43,7 +42,8 @@ describe('span should', () => {
             'test-service-name',
             reporter,
             new ConstSampler(true),
-            { logger: new MockLogger() }
+            { logger: new MockLogger() },
+            {},
```

Comment:
Its not clear to me what this parameter is.  Can we do something like "let upsamplingConfig = {}", also its been a while since I coded ES6, but this empty dict may refer to injectors, since that is the next parameter.